src/Management/test/EndpointBase.Test/ManagementEndpointOptionsTest.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright 2017 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+#pragma warning disable CS0612 // Type or member is obsolete
```

Comment:
marking this class with `[Obsolete]` should let the tests keep running and prevent the warning from showing